smithy-model/src/main/java/software/amazon/smithy/model/loader/IdlNodeParser.java

```diff
@@ -159,7 +163,7 @@ static String parseNodeObjectKey(IdlModelParser parser) {
         }
     }
 
-    private static ArrayNode parseArrayNode(IdlModelParser parser) {
+    static ArrayNode parseArrayNode(IdlModelParser parser) {
```

Comment:
Is this change needed? Or did I miss the new class where this is being used?

Reply:
I was using it at one point for the operation errors, but I ended up removing it. It's changed back now.